front/cypress/e2e/admin/pages_and_menu/update_homepage_content.cy.ts

```diff
@@ -0,0 +1,71 @@
+import { randomString } from '../../../support/commands';
+
+describe('Admin: update HomePage content', () => {
+  before(() => {
+    cy.setAdminLoginCookie();
+    cy.visit('/admin/pages-menu/');
+    cy.acceptCookies();
+    cy.wait(1000);
```

Comment:
No arbitrary waits should be needed in tests - please remove. If you want to wait for a specific api call to complete - use aliases to intercept / wait for it: https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Intercepts